README.md

```diff
@@ -43,6 +43,25 @@ This project provides a tool that simplifies secure shell connections over
    tab completion that will make life a lot easier for you.  Don't say we
    didn't warn you.
 
+## Nix ###
+
+If you have [Nix](https://nixos.org/download.html) installed you can use
+the [`flake.nix`](flake.nix) configuration file located at the root of the
+project to build and run `awssh`:
+
+```console
+nix build
+```
+
+After the build has completed, the `awssh` executable can be found at:
+`result/bin/awssh`
```

Comment:
If you are now generating a Nix `result/` directory in the project I would like it added to the `.gitignore` file so it is never accidentally committed.

Reply:
Please see 3106aa16ffbef91315e861171156b070fa6e2689

Reply:
And more recently, see https://github.com/cisagov/awssh/commit/c43cffbefe554146c7b5f2cbf137d26fd9a9358e.